RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHCAL_cfi.py

```diff
@@ -1,5 +1,10 @@
 import FWCore.ParameterSet.Config as cms
 
+_thresholdsHB = cms.vdouble(0.8, 0.8, 0.8, 0.8)
+_thresholdsHE = cms.vdouble(0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8)
+_thresholdsHBphase1 = cms.vdouble(0.1, 0.2, 0.3, 0.3)
+_thresholdsHEphase1 = cms.vdouble(0.1, 0.2, 0.2, 0.2, 0.2, 0., 0.2)
```

Comment:
NaNs are gone when the last but one value of this vector is fixed to be `0.2`

Reply:
I agree. This is supposed to be all 0.2, not 0..
It looks like the only ```0.,``` in this PR.